provider/doc/password-import.md

```diff
@@ -0,0 +1,10 @@
+You can import external passwords into your Pulumi programs as follows:
```

Comment:
I've replaced existing docs with this. 


```
name: test-random-1
runtime: yaml
description: A minimal Pulumi YAML program
resources:
  newPassword:
    type: random:RandomPassword
    properties:
      length: 11
      lower: true
      number: true
      numeric: true
      special: true
      upper: true
    options:
      protect: true
```

I wonder if it's worth calling out we have slightly too much code generated currently, but it does work. It does not have replace plans. We could improve this.

The upstream docs had a section on `Avoiding Replacement` that does not render correctly. I'm wondering how repro that in Pulumi land.